stanford_events.module

```diff
@@ -171,11 +171,23 @@ function stanford_events_node_view(array &$build, EntityInterface $entity, Entit
  *   The date formatted with stanford_long_day_date_year
  */
 function stanford_events_schedule_format_date($raw, $obj) {
-  // Expecting a timestamp in this function.
   if (!is_numeric($raw)) {
     return NULL;
   }
-  return \Drupal::service('date.formatter')->format($raw, 'stanford_long_day_date_year');
+
+  // We are going to use the passed object to determine if we are looking
+  // at an event that spans multiple days.
+  $schedule_date_time = $obj->su_schedule_date_time->get(0)->getValue();
```

Comment:
This isn't the correct way to grab the field value from an entity.
https://drupal.stackexchange.com/questions/144947/how-do-i-access-a-field-value-for-an-entity-e-g-node-object

Also you should always check for the field existing on the entity first.

Reply:
Added check to line 174, corrected field value access.